docs/community_tools.md

```diff
@@ -0,0 +1,62 @@
+# Community built tools around Jujutsu
+
+**Important:** Many of these are not complete yet, just like Jujutsu itself.
+But they already simplify many workflows and can improve your experience.
+
+
+## Diffedit3 
+
+Diffedit3 is a web-based alternate to Meld, as it no longer is packaged and 
+available for all Distros. Its creator is also a frequent contributor.
+
+Find it [here][diffedit3]
+
+## GG - GUI for JJ
+
+GG is a cross platform GUI for Jujutsu which makes all graph manipulating 
+workflows quite easy. Take a look at its README.md as it quite descriptive.
+
+Find it [here][gg].
+
+
+## LazyJJ
+
+lazyjj is a lazygit inspired TUI for Jujutsu.
+
+Find it [here][lazyjj].
+
+## JJ TUI
+
+This is TUI for Jujutsu built in Ocaml, it is un-opiniated and its creator is 
+open to feedback.
+
+Find it [here][jj_tui].
+
+
+## Visual Jujutsu
+
+VJJ is a fzf (fuzzy finder) wrapper for Jujutsu, which is meant to be used 
+interactively in the terminal. 
+
+Find it [here][vjj].
+
+## Cultivate
+
+Cultivate is an experimental backend implementation for Jujutsu, with the Goal 
+of integrating with a virtual file system like FUSE. As it is a major task, it
+still is very incomplete.
+
+Find it [here][cultivate].
```

Comment:
Is this functional enough to place it here? My opinion is that if we're linking to tools in the main documentation website, they should be functional, otherwise it might lead to confusion to new users.

Reply:
it definitely is to early, I removed it but left a TODO.